asgi.py

```diff
@@ -0,0 +1,4 @@
+from selectedtests.app.app import create_app
+from selectedtests.helpers import get_evg_api, get_mongo_wrapper
+
+app = create_app(get_mongo_wrapper(), get_evg_api())
```

Comment:
[nit] should this file be in the root of the project? 

We could put it in `src/selectedtests/app/app.py` basically replacing the `main` function there.